### PR TITLE
Tw/permgen

### PR DESCRIPTION
**Pull request by tweisser**
- Fix failed building (update to new AbstractAlgebra version)
- Add iterate for PrmGroup.

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -508,4 +508,13 @@ end
         @btime schreier_sims($(gens(G)))
         # GAP: 17 ms vs 50 ms here
     end
-end
\ No newline at end of file
+end
+
+@testset "Iterate over PrmGroup" begin
+	G = PermGroup(6);
+	K = PrmGroup(rand(G,2));
```
Comment (kalmarek) [inline test/runtests.jl]: use `K1 = PrmGroup([perm"(5,6)", perm"(1,2,3,4,5,6)"]) # Symmetric group on 6 symbols` 
and `@test order(K1) == length(uniq_elements) == 720`

add `K2 = PrmGroup([perm"(3,4,5)", perm"(1,2,3,4,5)"]) # Alternating group on 5 symbols`
of order `60`

File: test/runtests.jl [hunk 2]
```diff
@@ -508,4 +508,13 @@ end
...
+	elements = [g for g in K]
+	@test unique!(elements) == elements
```
Comment (kalmarek) [inline test/runtests.jl]: this tests nothing;
use `uniq_elements = unique(elements); @test uniq_elements == elements`

File: src/stabchain.jl [hunk 3]
```diff
@@ -120,3 +120,47 @@ end
...
+	goon = true
+	position = length(state) + 1
+	while goon && position > 1
+		position -= 1
+		t = transversals[position]
+		orbit = elements( t.orb )
+		if state[position] == orbit[end]
+			state[position] = orbit[1]
+		else
+			idx = findfirst(isequal(state[position]), orbit)
```
[the remark this answers is not in the document]
Comment (tweisser) [reply src/stabchain.jl]: I don't really see how to do this. Also, i couldn't iterate over transversals, when I tried to...

**Comment by tweisser** [comment]
using AbstractAlgebra
using PermutationGroups
G = PermGroup(6)
K = PrmGroup(rand(G,2))
bar = collect(K)
typeof(bar)
# Array{Any,1}

I don't understand this behavior. Every element of bar is Perm{int}....

File: src/stabchain.jl (hunk 3 above)
[the remark this answers is not in the document]
Comment (kalmarek) [reply src/stabchain.jl]: yes, I see that `Orbit1` does not allow for such iteration; I need to reevaluate benchmarks of orbits to see if it'd be worth switching to a different implementation, as they provide such `next`-style functionality

**Comment by kalmarek** [comment]
Seems ok to me; do you want to add something more here?

**Pull request merged**